pages/index.vue

```diff
@@ -57,50 +66,40 @@ async function fetchCategories() {
 
 async function fetchData(page: number = 1) {
   isLoading.value = true;
-  const apiUrl = "https://api.escuelajs.co/api/v1/products";
+  const offset = (page - 1) * ITEMS_PER_PAGE;
+  const apiUrl = `https://api.escuelajs.co/api/v1/products?offset=${offset}&limit=${ITEMS_PER_PAGE}`;
   const response = await fetch(apiUrl)
   items.value = await response.json()
   filteredItems.value = items.value;
-  paginatedItems.value = filteredItems.value.slice(0, itemsPerPage);
-  isLoading.value = false;
+  paginatedItems.value = filteredItems.value.slice(0, ITEMS_PER_PAGE);
```

Comment:
do we need it, if we are passing and offset in api

Reply:
Resolved

Reply:
Yes we need this

Reply:
Resolved